cygnus-ngsi/src/main/assembly/assemblyCompile.xml

```diff
@@ -0,0 +1,24 @@
+<assembly
```

Comment:
Could you elaborate on the purpose of this file, please?

Reply:
This file is a assembly which we use to add the ArcgisUtils-0.0.1-SNAPSHOT.jar to cygnus-ngsi.